inc/customizer/site-identity.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Additional site identity customizer options.
+ *
+ * @package Primer
+ * @since   NEXT
+ */
+
+class Primer_Site_Identity_Options {
+
+	/**
+	 * Class constructor.
+	 */
+	public function __construct() {
+
+		/**
+		 * Filter the site identity settings display.
+		 *
+		 * @since NEXT
+		 *
+		 * @var bool
+		 */
+		if ( ! (bool) apply_filters( 'primer_show_site_identity_settings', true ) ) {
+
+			return;
+
+		}
+
+		add_filter( 'primer_author_credit', array( $this, 'toggle_primer_author_credit' ) );
+
+		add_action( 'customize_register', array( $this, 'customize_register' ) );
+
+	}
+
+	/**
+	 * Toggle the visibility of the site credits in the footer.
+	 *
+	 * @action primer_author_credit
+	 * @since  NEXT
+	 *
+	 * @return bool
+	 */
+	public function toggle_primer_author_credit() {
+
+		return get_theme_mod( 'show_author_credit' ) ? true : false;
```

Comment:
@EvanHerman I think here it would be good to be explicit in what to consider false (e.g. `0`, `''`, `false`).

```
return ! empty( get_theme_mod( 'show_author_credit' ) );
```